library/testing/TestSDKEngineExceptions.cxx

```diff
@@ -24,9 +28,14 @@ int TestSDKEngineExceptions(int argc, char* argv[])
   }
   catch (const f3d::engine::exception& ex)
   {
-    std::cerr << ex.what() << std::endl;
-    exit(EXIT_SUCCESS);
+    std::cout << ex.what() << std::endl;
+    nbCatch++;
   }
 
-  return EXIT_FAILURE;
+  if (nbCatch != 2)
```

Comment:
If you look at my last changes for f3d::image test, I did something better. Just display a message and return in the `try` block.